config.ini

```diff
@@ -12,7 +7,7 @@ modules = telegram
 instance = mastodon.example.org
 
 # Mastodon user token.
-# Required permissions: read:statuses read:lists
+# Required permissions: read:accounts read:statuses read:lists
```

Comment:
Totally forgot about `read:accounts` permission, apparently we need it to get our own user info